src/index.js

```diff
@@ -192,7 +192,7 @@ class DropDownPicker extends React.Component {
 
         this.setState({
             isVisible: !this.state.isVisible,
-            direction: this.props.dropDownDirection || lowestPointOfDropdown < screenHeight ? 'top' : 'bottom',
+            direction: this.props.dropDownDirection ? this.props.dropDownDirection : lowestPointOfDropdown < screenHeight ? 'top' : 'bottom',
```

Comment:
Thanks for posting this! Out of curiosity if the type in `index.d.ts` is "top" or "bottom" now (per 2nd commit) how is this ternary working? It doesn't seem like it has a boolean to work with anymore?

Reply:
I've just updated this again. 
But here, `dropDownDirection` can be "top", "bottom" or undefined. And in this ternary, I am using the fact that "top" and "bottom" are truthy, and undefined is not.

Reply:
Ah I see! I must admit, I forget javascript truthiness when I'm first scanning things  - that's a bit subtle but also does look correct